config/locales/en.yml

```diff
@@ -885,6 +886,45 @@ en:
+          
+          Say you only want to track pages created from the “Historic sites templates” or pages which belong to the category “Algebra” but not “Geometry”. With the correct scoping method, doing this is incredibly easy
+        categories: Categories
+        categories_short_desc: Categories are intended to group together pages on similar subjects. Categories are normally found at the bottom of an article page.
+        categories_desc: >
+          Categories are intended to group together pages on similar subjects. Categories help readers to find, and navigate around, a subject area, to see pages sorted by title, and to thus find article relationships. 
+
+          You can enter the categories whose pages you wish to track. If the depth is set to 0, only the direct pages will be tracked. 
+        templates: Templates
+        templates_short_desc: Templates are pages intended to be included in other pages. Templates are normally found at the top of an article page.
+        petscan: PetScan
+        petscan_short_desc: PetScan is a tool that allows you to search for pages based on a wide range of criteria - pages in a category, pages that belong to a WikiProject, pages that have a certain template, etc
+        pagepile: PagePile
+        pagepile_short_desc: PagePile is a tool that allows you to create a list of pages which doesn't change over time and is identified by a unique ID
+        templates_desc: >
+          A template is a Wikipedia page created to be included in other pages. Templates usually contain repetitive material that might need to show up on a larger number of articles or pages. They are commonly used for boilerplate messages, standardized warnings or notices, infoboxes, navigational boxes, and similar purposes.
+        pagepile_desc: >
+          PagePile is a tool developed by the Wikimedia Foundation for the Wikimedia projects, including Wikipedia. It allows users to collect and organize a group of web page links into a single bundle called a "pile." These piles can then be easily shared with others, enabling collaborative work and information sharing.
+
+          PagePile serves as a convenient way to gather and present a curated set of Wikipedia articles or other web pages related to a specific topic or theme. It can be used for various purposes, such as creating reading lists, compiling research materials, organizing articles for a project, or simply bookmarking a collection of pages for personal reference
+        petscan_desc: >
+          PetScan is another tool developed by the Wikimedia Foundation that allows users to generate custom lists of Wikipedia pages based on specific criteria. It is a powerful tool for retrieving and filtering information from Wikipedia's vast database of articles.
```

Comment:
"PetScan is a powerful but complex tool for querying a set of wiki pages based on a wide variety of criteria and data sources. You can use PetScan directly to create the query for a set of articles you want to track, which gives you access to the full set of options to target your query very precisely. Alternatively, you can create a PetScan query from some of the most common options directly from the Dashboard.

PetScan saves the query parameters and provides an ID for that query. The results of that query may change over time, for example when new articles are added to category that the query includes. The Dashboard will periodically refresh the query results for a given ID, and will update its statistics based on the latest set of articles that PetScan produces."